### DISCUSSION TITLE
Drastically off from the expected value in LAMMPS

### DISCUSSION THREAD

**Discussion by hu-lm** [deepmodeling/deepmd-kit#1939] [Q&A]
Hi all,

I have recently used deepmd 2.1.3 to train several potential models of bcc iron as input for LAMMPS simulation. The NVT simulations in LAMMPS show a pretty good result as the predictions are close to the expected values. However, when LAMMPS performs the NPT simulations, the results start to be drastically off from the expected value (i.e the predicted density to be ~22 while the expected value for this is 14g/cc). I have tried to change several parameters like sel, training steps,  batch size, and learning rate. But this problem keeps occurring. I wonder if there is anything I could do that possibly can make my NPT simulations in LAMMPS more accurate? Thank you!

**Comment by wanghan-iapcm** [comment]
Does the volume/shape of simulation cell of your training data cover the target density and the possible fluctuation of the simulation cell in NPT simulations?

**Reply by hu-lm** [reply]
Hi,

Thank you for your reply. There was no change in the cell volume or shape in the training data (because they were based on vasp-NVT simulations). Also, are cell volume and shape information used during the training of the deep potential? Are we suggested to run iterative training by adding some vasp-NPT data into the training set? Thank you!

**Reply by wanghan-iapcm** [reply]
> Also, are cell volume and shape information used during the training of the deep potential?

Cell and virial information will be used when you turns on the virial prefactors (`start_pref_v` and `limit_pref_v`). This is recommended if the model will be used for NPT simulations. 

The configurations may also be sampled by NPT-DPMD as what  the DPGEN does.
